src/RA_StringUtils.h

```diff
@@ -441,49 +453,49 @@ class StringBuilder
             CharRef,
             WCharRef,
         };
-        Type DataType{ Type::String };
+        Type DataType{Type::String};
     };
 
     mutable std::vector<PendingString> m_vPending;
-};
+}; // namespace ra
 
 template<typename... Ts>
-void AppendString(std::string& sString, Ts... args)
+void AppendString(std::string& sString, const Ts... args)
```

Comment:
if it's not possible to put `Ts&&... args`, at least get rid of the `const`.

Reply:
`const Ts&&` doesn't like string literals:
```
cannot convert argument 2 from 'const char [7]' to 'const char (&&)[7]'
```
But I was able to make it work using `Ts&&` as the input and forwarding them as `const`.